data/stoa0171/stoa001/__cts__.xml

```diff
@@ -1,7 +1,8 @@
 <ti:work xml:lang="lat" xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:latinLit:stoa0171" urn="urn:cts:latinLit:stoa0171.stoa001">
-  <ti:title xml:lang="eng">De ave phoenice</ti:title>
-  <ti:edition workUrn="urn:cts:latinLit:stoa0171.stoa001" urn="urn:cts:latinLit:stoa0171.stoa001.opp-lat3">
-    <ti:label xml:lang="eng">De ave phoenice</ti:label>
-    <ti:description xml:lang="eng">Lactantius ca. 240-ca. 320, De ave phoenice</ti:description>
+  <ti:title xml:lang="lat">De ave phoenice</ti:title>
+  <ti:edition xml:lang="lat" workUrn="urn:cts:latinLit:stoa0171.stoa001" urn="urn:cts:latinLit:stoa0171.stoa001.opp-lat3">
+    <ti:label xml:lang="lat">De ave phoenice</ti:label>
+    <ti:description xml:lang="mul">Lactantius ca. 240-ca. 320. De ave phoenice. L. Caeli Firmiani Lactanti Opera omnia, Part II, Fasc 1. 
```

Comment:
We don't need the years or the work title here (unless they appear on the title page of the book). It keeps it lighter and more readable.
It looks like the title is L. Caeli Firmiani Lactanti Opera omnia. 
It also looks like there are 2 editors? George Laubmann is also listed.

Reply:
Oh yea, I missed the second editor when I reviewed the catalog record, sigh. This is why I'm having you review these first ones, I'm hoping as I do more I'll make fewer mistakes. Thanks!